examples/appprotect-dos/apdos-protected.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: appprotectdos.f5.com/v1beta1
+kind: DosProtectedResource
+metadata:
+  name: dos-protected
+spec:
+  enable: true
+  name: "webapp.example.com"
+  apDosPolicy: "dospolicy"
+  apDosMonitor:
+    uri: "webapp.example.com"
+    protocol: "http1"
+    timeout: 5
+  dosAccessLogDest: "127.0.0.1:5561"
```

Comment:
@pasmant ^

Reply:
the access logs are different from the other logs, as far as I understand, they're essentially internal, hence the 127.0.0.1

Reply:
@soneillf5 
I think this can cause confusion, because we set that setting, but don't really show how to use it. So it's left for the reader to figure that out.

More over, we set the setting to something very specific - `127.0.0.1:5561`. This assumes that inside the IC container there is a syslog log collector running on that address:port. Which isn't the case afaik. 

I'd suggest updating this setting in way that the reader (user) would be able to access the logs.

Reply:
removed